Formula/a/ampl-mp.rb

```diff
@@ -23,49 +22,33 @@ class AmplMp < Formula
 
   depends_on "cmake" => :build
 
-  resource "miniampl" do
-    url "https://github.com/dpo/miniampl/archive/refs/tags/v1.0.tar.gz"
-    sha256 "b836dbf1208426f4bd93d6d79d632c6f5619054279ac33453825e036a915c675"
-  end
-
-  # Removes Google Benchmark - as already done so upstream
-  # All it did was conflict with the google-benchmark formula
-  patch do
-    url "https://github.com/ampl/mp/commit/96e332bb8cb7ba925e3ac947d6df515496027eed.patch?full_index=1"
-    sha256 "1a4ef4cd1f4e8b959c20518f8f00994ef577e74e05824b2d1b241b1c3c1f84eb"
-  end
-
-  # Install missing header files, remove in > 3.1.0
-  # https://github.com/ampl/mp/issues/110
```

Comment:
Descriptions of all 3 pathches were not copied over to `ampl-mp@3.rb`. 
Was it intentional?
IMO it is not ok to have patches without some description.

Reply:
agree, I think I forgot to copied while tweaking the versioned formula for ipopt, anyway after having upstream released 1.0.0 for ampl-asl, we no longer need ampl-mp@3